### PR TITLE
Fix: Statistics design changes

### PR DESCRIPTION
**Pull request by bstien**
# Why?
The icon for clicks should be changed, the title of the value should be made bold and the icon should be 24x24.

# What?
- 👆 Fix these.
- Remove unneeded public access from `StatisticsItemView`.

# Version Change
Breaking. Making public view internal.

# UI Changes
| Before | After |
| --- | --- |
| Screenshot 2023-02-07 at 11 44 22 | Screenshot 2023-02-07 at 11 55 26 |

### CODE REVIEW

File: FinniversKit/Sources/Components/StatisticsView/StatisticsItemView.swift [hunk 1]
```diff
@@ -4,37 +4,38 @@
 
 import UIKit
 
-public class StatisticsItemView: UIView {
+class StatisticsItemView: UIView {
```
Comment (BjornRuud) [inline FinniversKit/Sources/Components/StatisticsView/StatisticsItemView.swift:7]: Is the class used in inheritance or can it be final?

File: FinniversKit/Sources/Assets/Icons.xcassets/AdManagement/statsClick.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "filename" : "statsClick.pdf",
```
Comment (BjornRuud) [inline FinniversKit/Sources/Assets/Icons.xcassets/AdManagement/statsClick.imageset/Contents.json:4]: Is this icon available as SVG? I think we should prefer SVG when possible as it is text-based. It should also have the preserve vector data flag set.

**Review by BjornRuud — APPROVED** [review]
Looks good! 👍

File: FinniversKit/Sources/Components/StatisticsView/StatisticsItemView.swift (hunk 1 above)
Comment (bstien) [reply FinniversKit/Sources/Components/StatisticsView/StatisticsItemView.swift:7]: Good catch – adding

**Pull request merged**